root_VS2012/programs/C#/Samples/WebApp_sample/MVC_Sample/MVC_Sample/Controllers/PingController.cs

```diff
@@ -0,0 +1,43 @@
+﻿//**********************************************************************************
+//* Ping Controller
+//**********************************************************************************
+
+//**********************************************************************************
+//* クラス名        ：PingController
+//* クラス日本語名  ：Ping Controller for Html.BeginForm
+//*
+//* 作成日時        ：－
+//* 作成者          ：sas 生技
+//* 更新履歴        ：
+//*
+//*  日時        更新者            内容
+//*  ----------  ----------------  -------------------------------------------------
+//*  2015/12/28  Sai               Added controller to prevent session timeout.
+//*  2016/01/12  Sai               Removed method Ping and modified Index method to return empty string.
+//*  2016/01/12  Sai               Modfied XML comments.
+//**********************************************************************************
+
+//System
+using System.Web.Mvc;
+
+namespace MVC_Sample.Controllers
+{
+    /// <summary>
+    /// Ping Controller for Html.BeginForm
```

Comment:
@yusukemaegawa 

This is because the view that corresponds to this controller is implemented by Html.BeginForm.